content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+
+* `/redfish/v1/JsonSchemas`
+* `/redfish/v1/Registries`
+
+These URIs contain links to all schemas and registries used by the service. They point to documents stored in the BMC, if it has the storage capacity, or to the official DMTF website. In the later case, Redfish® clients must have a mean to access the Internet to download those documents. They are be helpful to identify property interdependencies, supported values or read-write capability.
+
+### Allowed requests
+
+The Redfish® protocol requires GET request responses to contain the `Allow` header indicating the possible operations on the URI. Figure 9 shows that the URI `/redfish/v1/Chassis/{ChassisId}` allows GET, HEAD, and PATCH operations against this URI.
+
+![Figure 9: Possible operations on the URI](/img/fig9-allowheader.png "Figure 9: Possible operations on the URI")
+
+After verification, the client code can change (PATCH) the properties marked as `ReadOnly=False` in the schema. The main server chassis in Figure 9 shows the PATCH method and the IndicatorLED property can be modified (Figure 10), so we should be able to turn on the small blue LED on this chassis to better identify it in the data center. A good programmer will perform all of these checks before sending their changes. The bad programmer will not perform any checks and will leave the end user to deal with the error(s) returned by the service!
+
+### Deprecated resources
+
+Over time, certain properties are deprecated. This information is mentioned in the schemas. Thus, clients receiving a “property not found” response will be able to consult the schema and check if by chance this property has not been deprecated. This type of verification will avoid comments from the programmer like "I don’t understand, it worked before!". But before what? Figure 10 shows an excerpt from the `#Chassis` schema, version 1.22.0, stating that the `IndicatorLED` property has been deprecated in version 1.14.0.
+
+![Figure 10: Deprecated property](/img/fig10-propertydeprecated.png "Figure 10: Deprecated property")
+
+## What else?
+
+The separation of protocol from data modeling, collections, OEM extensions and registries form the basis of the Redfish® standard. Access to schemas and registries as well as the list of possible operations for each URI are important features allowing the writing of portable client code in time and space.
+
+However, Redfish® is full of other particularities, conceptual and practical, which are very interesting to study. For example the mode of communication with additional storage or network cards, updating firmware, or even the use of Swordfish® schemas developed by the Storage and Network Industry Association (<a href="https://www.snia.org/" target="_blank">SNIA</a>). Many security aspects are also addressed by Redfish® such as the integrity of the various server components. Read the second part of this Redfish® introduction for a complete view of this standard API.
```

Comment:
Hmmm... that second sentence isn't a complete sentence... in spite of how long it is.  You need a subject and a verb...